### PR TITLE
Adds the travis config

### PR DESCRIPTION
**Pull request by lfac-pt**

### CODE REVIEW

**Comment by victorfern91** [comment]
Unit tests are falling 😞

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "test": "jest --coverage",
     "test:watch": "jest --watch",
-    "lint": "eslint src/**/*.js",
+    "lint": "eslint src/{,**/}*.js",
```
[the remark this answers is not in the document]
Comment (lfac-pt) [reply package.json]: At least in my mac that won't lint the files directly inside `src` only the ones nested a level deeper.

[the remark this answers is not in the document]
Comment (victorfern91) [reply package.json]: Ok

File: src/utils.test.js [hunk 2]
```diff
@@ -12,6 +12,18 @@ import { smallSample } from "../stories/sampleData";
 
 let xAccessor, yAccessor, previousBrushDomain;
 
+// We are doing this because `timeMonth` returns dates with an 1 hour offset
```
Comment (victorfern91) [inline src/utils.test.js]: We could move this mock to an isolated file in the `__mock__` folder

File: src/Histogram/Histogram.test.js [hunk 3]
```diff
@@ -3,14 +3,31 @@ import { timeFormat } from "d3-time-format";
 import { mount } from "enzyme";
 import { Histogram } from "./Histogram";
 
-import { smallSample } from "../stories/sampleData";
+import { smallSample } from "../../stories/sampleData";
 
-jest.mock("./canvasRenderUtils", () => ({
+jest.mock("../canvasRenderUtils", () => ({
```
Comment (victorfern91) [inline src/Histogram/Histogram.test.js]: We could move this mock into a file

**Pull request merged**